custom_components/localtuya/pytuya/__init__.py

```diff
@@ -384,52 +364,27 @@ def status(self):
             log.error('Unexpected status() payload=%r', result)
 
         return result
-
-    def set_status(self, on, switch=1):
-        """
-        Set status of the device to 'on' or 'off'.
-        
-        Args:
-            on(bool):  True for 'on', False for 'off'.
-            switch(int): The switch to set
-        """
-        # open device, send request, then close connection
-        if isinstance(switch, int):
-            switch = str(switch)  # index and payload is a string
-        payload = self.generate_payload(SET, {switch:on})
-        #print('payload %r' % payload)
-
-        data = self._send_receive(payload)
-        log.debug('set_status received data=%r', data)
-
-        return data
     
-    def set_value(self, index, value):
+    def set_dps(self, value, dpsIndex):
```

Comment:
Snake case is used in python (and the rest of this file), so I suggest `dps_index` instead.

Reply:
Agree, done